src/sources/prometheus/mod.rs

```diff
@@ -334,3 +333,69 @@ mod test {
         topology.stop().compat().await.unwrap();
     }
 }
+
+#[cfg(all(test, feature = "prometheus-integration-tests"))]
+mod integration_tests {
+    use super::*;
+    use crate::{
+        event::{MetricKind, MetricValue},
+        shutdown, test_util, Pipeline,
+    };
+    use futures::compat::Future01CompatExt as _;
```

Comment:
Out of interest, what purpose of `use X as _;`? I understand that this can solve the issue when we import different traits with the same name (like `futures01::Stream` and `futures::Stream`), but here we do not have any other `Future01CompatExt` :thinking:

Reply:
I've been starting to do this too when importing traits solely for the side-effect of having them in-scope. I think it helps indicate that the import is a trait that is never used directly.

Reply:
I picked up this habit from https://tarquin-the-brave.github.io/blog/posts/rust_use_statements/

Reply:
@jszwedko hit it on the nose, on both counts.